pkg/api/bzz_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package api_test
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/ethersphere/bee/pkg/api"
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+	"github.com/ethersphere/bee/pkg/jsonhttp/jsonhttptest"
+	"github.com/ethersphere/bee/pkg/storage/mock"
+)
+
+func TestBzz(t *testing.T) {
+	var (
+		resource        = "/bzz/"
+		content         = []byte("foo")
+		expHash         = "b9d678ef39fa973b430795a1f04e0f2541b47c996fd300552a1e8bfb5824325f"
```

Comment:
as you correctly spotted this is not correct hash when using correct segment count in bmt.